app/src/components/GrantCard.vue

```diff
@@ -94,6 +96,8 @@ export default defineComponent({
     const grantId = ref<number>(props.id);
     const raised = computed(() => getTotalRaised(grantId.value));
     const { addToCart, isInCart, removeFromCart } = useCartStore();
+    const { approvedGrantsPk } = useDataStore();
+    const idList = computed(() => approvedGrantsPk.value || []);
```

Comment:
`approvedGrantsPk` is already computed when returned from the store, so we don't need to re-wrap it in `computed` here. Let's just return `approvedGrantsPk` a few lines down

Reply:
I see, it was throwing linter errors so might still need the variable `idList` think it's bc approvedGrantsPk could be null

Reply:
just saw your comments below and will do that instead